hello_world/main.py

```diff
@@ -0,0 +1,102 @@
+#!/usr/bin/env python
+
+from typing import Any
+
+import pkg_resources
+from helpers import make_table
+from prompt_toolkit.application import Application
+from prompt_toolkit.application.current import get_app
+from prompt_toolkit.key_binding import KeyBindings
+from prompt_toolkit.key_binding.bindings.focus import focus_next, focus_previous
+from prompt_toolkit.layout import HSplit, Layout, VSplit
+from prompt_toolkit.styles import Style
+from prompt_toolkit.widgets import Box, Button, Label, TextArea
+
+
+# Event handlers for all the buttons.
+def get_pkg_info() -> str:
+    """
+    Function to give the package information in form of  a table.
```

Comment:
```suggestion
    Function to give the package information in form of a table.
```
Like i told joe, `Function` can be ignored since we can understand that by the definition.

Reply:
Alright